src/aselo-webchat.tsx

```diff
@@ -85,8 +88,8 @@ export const initWebchat = async () => {
   if (currentConfig.captureIp) {
     ip = await getUserIp();
   }
-
-  if (Array.isArray(blockedIps) && ip && (<string[]>blockedIps).includes(ip)) {
+// @ts-ignore
```

Comment:
I had to make this change I am getting a bunch of ts errors (possibly because blockedIps a json format??) 
The above is my work around. I am wondering if there is a better approach

Reply:
`// @ts-ignore` is always the very last resort, and should only be used to address things that can't be addressed any other way, most straightforward TS errors can be worked around by casting

A specific gotcha in tsx files is that you can't cast things using the `<myType>myVar` syntax, because the parser thinks <> denotes a JSX tag, you always have to cast using the `myVar as myType` form

One of the many reasons I hate JSX - everyone should use hyperscript! :-P